virtcontainers/sandbox.go

```diff
@@ -925,6 +929,172 @@ func (s *Sandbox) removeNetwork() error {
 	return nil
 }
 
+// AddNetwork adds new nic to the sandbox.
+func (s *Sandbox) AddNetwork(ifName string) error {
```

Comment:
I would prefer the API to be declarative so that the interface is configured explicitly at the API level, e.g., to let caller to set bridge, ip, mac, mtu, gw etc through the API, instead of letting virtcontainers discover what's there through netns. This also aligns with `UpdateRoute()`. @caoruidong @sboeuf @WeiZhang555 WDYT?

Reply:
Agree. I will change my code.